docs/concepts/effective-gno.md

```diff
@@ -425,7 +492,7 @@ The goal is usually to store the admin address or a list of addresses
 owners. These helper functions should check if the caller of a function is
 whitelisted or not.
 
-Let's deep dive into the different access control mechanisms we can use:
+Lets deep dive into the different access control mechanisms we can use:
```

Comment:
This is a typo, the correct form was `Let's`

Reply:
Ah, you're right, don't know what was going through my head. Fixed it: [3091d76](https://github.com/gnolang/gno/pull/2047/commits/3091d76e2c667d9bbd300a0af5cc539f87d8a9df)